src/PrestaShopBundle/Form/Admin/Type/ButtonCollectionType.php

```diff
@@ -78,20 +84,8 @@ public function configureOptions(OptionsResolver $resolver)
                 'label' => false,
                 'buttons' => [],
                 'justify_content' => 'space-between',
-                // Internal option automatically built based on the group option of each button
-                'button_groups' => [],
             ])
             ->setAllowedTypes('buttons', 'array')
-            ->setNormalizer('button_groups', function (Options $options) {
```

Comment:
Why do you remove this normalizer and not the other one ? :thinking:

Reply:
I removed this because I realized I was introducing a `button_groups` which was only used internally So it complexifies the form type contract for nothing, it contained an option that was not editable and it relied on a normalizer which is convenient in some cases but not easy to grasp for everyone

So I preferred to simplify this code and handle the formatting of the `buttons` option locally in the buildView method, I also could have used this normalizer for the `buttons` option it would work but as I said it would bring a bit of complexity that is not really required here